src/validator.js

```diff
@@ -0,0 +1,52 @@
+const { get, isNil } = require("lodash");
+const buildRoutingGraph = require("../src/helpers/build-routing-graph");
+const detectOrphan = require("./features/routing/detect-orphan");
+
+const failFast = errors => {
+  if (errors.errors.length > 0) {
+    throw new Error(JSON.stringify(errors, null, 2));
+  }
+};
+
+//eslint-disable-next-line
+const validatePage = (page, ctx, errors, options) => {
+  //Call your page-level validation scripts here
+  const graph = buildRoutingGraph(ctx);
+  const orphanError = detectOrphan(page.id, ctx, graph);
+  if (!isNil(orphanError)) errors.errors.push(orphanError); //eslint-disable-line
```

Comment:
Probably should just put the curlys in. Or you could use the ternary instead.

Reply:
That's fair. I'll use the ternary.

Reply:
Ternary threw an error also with the linter as it expected a function or assignment, so I used curlys.